sdk/communication/azure-communication-phonenumbers/test/sip_routing_helper.py

```diff
@@ -0,0 +1,38 @@
+# ------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+
+import os
+
+from devtools_testutils import is_live
+
+def get_user_domain():
+    if(is_live()):
+        sip_domain = os.getenv("AZURE_TEST_SIP_DOMAIN")
```

Comment:
From what I understood in the PR description, the value of the `AZURE_TEST_SIP_DOMAIN` variable should be a randomly generated UUID. Did you consider creating it at runtime instead of using an environment variable?

Reply:
Yes, actually, we had conversation with Lucas on this topic, because originally, the domain was generated at runtime, but Lucas suggested to use environment variable instead. I can revert it back to UUID.

Reply:
Got it, I wasn't aware of that previous discussion. There's nothing wrong with having the environment variable, it just requires more effort since we need to make sure that the pipeline job is created with a new random value for it on every run. My comment was purely to understand the reasoning behind the decision and raise awareness of said additional effort. However, since you already reverted it, I think we are good to go.